Build break: Windows app build failed on Windows 2022 GitHub image

[Example run](https://github.com/mattermost/focalboard/runs/5203459175?check_suite_focus=true)

## Build error logs

```
Build started 2/15/2022 4:39:38 PM.
Project "D:\a\focalboard\focalboard\win-wpf\Focalboard.sln" on node 1 (Rebuild target(s)).
ValidateSolutionConfiguration:
  Building solution configuration "Release|x64".
Project "D:\a\focalboard\focalboard\win-wpf\Focalboard.sln" (1) is building "D:\a\focalboard\focalboard\win-wpf\Focalboard\Focalboard.csproj" (2) on node 1 (Rebuild target(s)).
CoreClean:
  Creating directory "obj\x64\Release\".
PrepareForBuild:
  Creating directory "bin\x64\Release\".
ResolveAssemblyReferences:
  Primary reference "Windows".
C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\Microsoft.Common.CurrentVersion.targets(2304,5): warning MSB3245: Could not resolve this reference. Could not locate the assembly "Windows". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors. [D:\a\focalboard\focalboard\win-wpf\Focalboard\Focalboard.csproj]
          For SearchPath "{CandidateAssemblyFiles}".
          Considered "D:\a\focalboard\focalboard\win-wpf\packages\Microsoft.Web.WebView2.1.0.705.50\build\\..\runtimes\win-x86\native\WebView2Loader.dll",
  			but its name "WebView2Loader"
  			didn't match the expected name "Windows".
          Considered "D:\a\focalboard\focalboard\win-wpf\packages\Microsoft.Web.WebView2.1.0.705.50\build\\..\runtimes\win-x64\native\WebView2Loader.dll",
  			but its name "WebView2Loader"
  			didn't match the expected name "Windows".
          Considered "D:\a\focalboard\focalboard\win-wpf\packages\Microsoft.Web.WebView2.1.0.705.50\build\\..\runtimes\win-arm\native\WebView2Loader.dll",
  			but its name "WebView2Loader"
  			didn't match the expected name "Windows".
          For SearchPath "{HintPathFromItem}".
          Considered "C:\Program Files (x86)\Windows Kits\10\UnionMetadata\10.0.18362.0\Windows.winmd", but it didn't exist.
          For SearchPath "{TargetFrameworkDirectory}".
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\Windows.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\Windows.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\Windows.exe", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\Facades\Windows.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\Facades\Windows.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\Facades\Windows.exe", but it didn't exist.
          For SearchPath "{AssemblyFoldersFromConfig:C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\AssemblyFolders.config,v4.7.2}".
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\Microsoft\SqlDb\Windows.winmd", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\Microsoft\SqlDb\Windows.dll", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\Extensions\Microsoft\SqlDb\Windows.exe", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\PublicAssemblies\Windows.winmd", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\PublicAssemblies\Windows.dll", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\PublicAssemblies\Windows.exe", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.5\Windows.winmd", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.5\Windows.dll", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.5\Windows.exe", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.0\Windows.winmd", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.0\Windows.dll", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v4.0\Windows.exe", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v2.0\Windows.winmd", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v2.0\Windows.dll", but it didn't exist.
          Considered "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Common7\IDE\ReferenceAssemblies\v2.0\Windows.exe", but it didn't exist.
          For SearchPath "{Registry:Software\Microsoft\.NETFramework,v4.7.2,AssemblyFoldersEx}".
          Considered AssemblyFoldersEx locations.
          For SearchPath "{AssemblyFolders}".
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.0\Windows.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.0\Windows.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.0\Windows.exe", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.5\Windows.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.5\Windows.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\v3.5\Windows.exe", but it didn't exist.
          Considered "C:\Program Files\IIS\Microsoft Web Deploy V3\Windows.winmd", but it didn't exist.
          Considered "C:\Program Files\IIS\Microsoft Web Deploy V3\Windows.dll", but it didn't exist.
          Considered "C:\Program Files\IIS\Microsoft Web Deploy V3\Windows.exe", but it didn't exist.
          Considered "C:\Program Files (x86)\Microsoft SQL Server\130\SDK\Assemblies\Windows.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\Microsoft SQL Server\130\SDK\Assemblies\Windows.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\Microsoft SQL Server\130\SDK\Assemblies\Windows.exe", but it didn't exist.
          Considered "C:\Program Files (x86)\WiX Toolset v3.11\SDK\Windows.winmd", but it didn't exist.
          Considered "C:\Program Files (x86)\WiX Toolset v3.11\SDK\Windows.dll", but it didn't exist.
          Considered "C:\Program Files (x86)\WiX Toolset v3.11\SDK\Windows.exe", but it didn't exist.
          For SearchPath "{GAC}".
          Considered "Windows", which was not found in the GAC.
          For SearchPath "{RawFileName}".
          Considered treating "Windows" as a file name, but it didn't exist.
          For SearchPath "bin\x64\Release\".
          Considered "bin\x64\Release\Windows.winmd", but it didn't exist.
          Considered "bin\x64\Release\Windows.dll", but it didn't exist.
          Considered "bin\x64\Release\Windows.exe", but it didn't exist.
GenerateBindingRedirects:
  No suggested binding redirects from ResolveAssemblyReferences.
Project "D:\a\focalboard\focalboard\win-wpf\Focalboard\Focalboard.csproj" (2) is building "D:\a\focalboard\focalboard\win-wpf\Focalboard\Focalboard_d4yzzl4r_wpftmp.csproj" (3) on node 1 (_CompileTemporaryAssembly target(s)).
CoreCompile:
  C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\Roslyn\csc.exe /noconfig /nowarn:[170](https://github.com/mattermost/focalboard/runs/5203459175?check_suite_focus=true#step:12:170)1,1702 /fullpaths /nostdlib+ /platform:x64 /errorreport:prompt /warn:4 /define:TRACE /highentropyva+ /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\Microsoft.CSharp.dll" /reference:D:\a\focalboard\focalboard\win-wpf\packages\Microsoft.Web.WebView2.1.0.705.50\lib\net45\Microsoft.Web.WebView2.Core.dll /reference:D:\a\focalboard\focalboard\win-wpf\packages\Microsoft.Web.WebView2.1.0.705.50\lib\net45\Microsoft.Web.WebView2.WinForms.dll /reference:D:\a\focalboard\focalboard\win-wpf\packages\Microsoft.Web.WebView2.1.0.705.50\lib\net45\Microsoft.Web.WebView2.Wpf.dll /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\mscorlib.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\PresentationCore.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\PresentationFramework.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\System.Core.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\System.Data.DataSetExtensions.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\System.Data.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\System.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\System.Drawing.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\System.Net.Http.dll" /reference:C:\Windows\Microsoft.Net\assembly\GAC_MSIL\System.Runtime.WindowsRuntime\v4.0_4.0.0.0__b77a5c56[193](https://github.com/mattermost/focalboard/runs/5203459175?check_suite_focus=true#step:12:193)4e089\System.Runtime.WindowsRuntime.dll /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\System.Xaml.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\System.Xml.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\System.Xml.Linq.dll" /reference:"C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\WindowsBase.dll" /debug- /filealign:512 /optimize+ /out:obj\x64\Release\Focalboard.exe /subsystemversion:6.00 /target:winexe /utf8output /win32icon:focalboard.ico /deterministic+ /langversion:7.3 /analyzerconfig:D:\a\focalboard\focalboard\.editorconfig Utils.cs Webview2Installer.cs App.xaml.cs MainWindow.xaml.cs Properties\AssemblyInfo.cs Properties\Resources.Designer.cs Properties\Settings.Designer.cs D:\a\focalboard\focalboard\win-wpf\Focalboard\obj\x64\Release\MainWindow.g.cs D:\a\focalboard\focalboard\win-wpf\Focalboard\obj\x64\Release\App.g.cs D:\a\focalboard\focalboard\win-wpf\Focalboard\obj\x64\Release\Focalboard_Content.g.cs
D:\a\focalboard\focalboard\win-wpf\Focalboard\Webview2Installer.cs(9,15): error CS0234: The type or namespace name 'Storage' does not exist in the namespace 'Windows' (are you missing an assembly reference?) [D:\a\focalboard\focalboard\win-wpf\Focalboard\Focalboard_d4yzzl4r_wpftmp.csproj]
D:\a\focalboard\focalboard\win-wpf\Focalboard\App.xaml.cs(13,15): error CS0234: The type or namespace name 'Storage' does not exist in the namespace 'Windows' (are you missing an assembly reference?) [D:\a\focalboard\focalboard\win-wpf\Focalboard\Focalboard_d4yzzl4r_wpftmp.csproj]
  CompilerServer: server - server processed compilation - b1c7a2d4-cadc-48ee-9a8d-835f9a9e7f18
Done Building Project "D:\a\focalboard\focalboard\win-wpf\Focalboard\Focalboard_d4yzzl4r_wpftmp.csproj" (_CompileTemporaryAssembly target(s)) -- FAILED.
```

## Additional context

Looks like some dependencies have changed in the Windows 2022 server image

--- a/.github/workflows/dev-release.yml
+++ b/.github/workflows/dev-release.yml
@@ -111,7 +111,7 @@ jobs:
         path: ${{ github.workspace }}/mac/dist/focalboard-mac.zip
 
   windows:
-    runs-on: windows-latest
+    runs-on: windows-2019
 
     steps:
     - name: Checkout

--- a/.github/workflows/prod-release.yml
+++ b/.github/workflows/prod-release.yml
@@ -106,7 +106,7 @@ jobs:
         path: ${{ github.workspace }}/mac/dist/focalboard-mac.zip
 
   windows:
-    runs-on: windows-latest
+    runs-on: windows-2019
 
     steps:
     - name: Checkout

